app/controllers/users_controller.rb

```diff
@@ -34,22 +34,23 @@ class UsersController < ApplicationController
 
   def index
     @user = User.find(current_user.id)
-    @slides = Slide.
-              latest.
-              owner(current_user.id).
-              includes(:user).
-              paginate(page: params[:page], per_page: 30)
+    @slides = user_slide_with_paginate(current_user.id)
     @tags = Slide.
             owner(current_user.id).
             tag_counts_on(:tags).order('count DESC')
   end
 
   def show
     @user = User.find(params[:id])
-    @slides = Slide.published.latest.
-              owner(params[:id]).
-              includes(:user).
-              paginate(page: params[:page], per_page: 30)
+    @slides = user_slide_with_paginate(params[:id])
+    # if params[:sort_by] == 'popularity'
+      # @slides = Slide.published.popular
+    # else
```

Comment:
__[rubocop]__
* Style/CommentIndentation: Incorrect indentation detected (column 4 instead of 6).